src/storage.cc

```diff
@@ -184,9 +199,21 @@ Status Storage::Open(bool read_only) {
   metadata_opts.disable_auto_compactions = config_->RocksDB.disable_auto_compactions;
   metadata_opts.table_properties_collector_factories.emplace_back(
       NewCompactOnExpiredTableCollectorFactory(kMetadataColumnFamilyName, 0.3));
+  // BlobDB
+  metadata_opts.enable_blob_files = config_->RocksDB.enable_blob_files;
+  metadata_opts.min_blob_size = config_->RocksDB.min_blob_size;
+  metadata_opts.blob_file_size = config_->RocksDB.blob_file_size;
+  metadata_opts.blob_compression_type = static_cast<rocksdb::CompressionType>(config_->RocksDB.compression);
+  metadata_opts.enable_blob_garbage_collection = config_->RocksDB.enable_blob_garbage_collection;
+  metadata_opts.blob_garbage_collection_age_cutoff = config_->RocksDB.blob_garbage_collection_age_cutoff / 100;
 
   rocksdb::BlockBasedTableOptions subkey_table_opts;
-  subkey_table_opts.filter_policy.reset(rocksdb::NewBloomFilterPolicy(10, true));
+  subkey_table_opts.format_version = 5;
```

Comment:
I find metadata CF's some options are the same as and subkey CF, could we create a new function to set both of them options to make code clean

Reply:
Good idea